frontends/mit-open/src/pages/FieldPage/FieldPage.test.tsx

```diff
@@ -105,7 +115,19 @@ describe("FieldPage", () => {
     expect(images[0].src).toContain(field.configuration.banner_background)
     expect(images[1].src).toContain(field.configuration.logo)
   })
+  it("Displays a featured carousel if the channel type is 'offeror'", async () => {
+    const { field } = setupApis({
+      search_filter: "offeror=ocw",
+      channel_type: "offeror",
+    })
 
+    renderTestApp({ url: `/c/${field.channel_type}/${field.name}` })
+    await screen.findAllByText(field.title)
+    const carousel = await screen.findByText("Featured Courses")
+    act(() => {
+      expect(carousel).toBeInTheDocument()
+    })
```

Comment:
```suggestion
    expect(carousel).toBeInTheDocument()
```
Not need for act here.

Actually, the assertion doesn't really add anything—`findByText` will error if it's not found. But fine to keep the assertion if you like it... It is more explicit.

Reply:
my precommit or lint check was insisting on including the act for some reason. in any case i'll leave for now